upi/metal/main.tf

```diff
@@ -8,13 +8,15 @@ provider "ignition" {
 # ================MATCHBOX=====================
 
 locals {
+  initrd_list = split("/", var.pxe_initrd_url)
   kernel_args = [
     "console=tty0",
     "console=ttyS1,115200n8",
     "rd.neednet=1",
 
     # "rd.break=initqueue"
     "coreos.inst=yes",
+    "initrd=${ local.initrd_list[ length(local.initrd_list) - 1 ] }",
```

Comment:
Can you explain a little why this change is required?

Reply:
Without this, the c2.medium Packet instances would not boot, because they could not find the initramfs
https://gist.github.com/mulbc/eb4b82e5ea0d1c685e67f62af29db84e

With this, they booted just fine

Reply:
we run CI tests with current configuration https://prow.ci.openshift.org/view/gcs/origin-ci-test/logs/release-openshift-ocp-installer-e2e-metal-4.6/1296164699951337472 and it seems to be working fine,

Can you explain a little bit more exactly what error msg / failure you are seeing.

Reply:
Some packet instances don't not without this.
Since the Terraform code is hard coded to use small instances, you won't see the differences in the CI, but only when someone chooses different (larger) instances